src/tests/ftest/soak/soak_test_base.py

```diff
@@ -24,17 +24,18 @@
 
 import os
 import time
+import threading
 from apricot import TestWithServers
 from general_utils import run_command, DaosTestError, get_log_file
 import slurm_utils
 from ClusterShell.NodeSet import NodeSet
 from getpass import getuser
 import socket
 from agent_utils import include_local_host
-
 from soak_utils import DDHHMMSS_format, add_pools, get_remote_logs, \
-    is_harasser, launch_harassers, harasser_completion, create_ior_cmdline, \
-    cleanup_dfuse, create_fio_cmdline, build_job_script, SoakTestError
+    launch_rebuild, launch_snapshot, launch_exclude_reintegrate, \
+    create_ior_cmdline, cleanup_dfuse, create_fio_cmdline, \
+    build_job_script, SoakTestError
```

Comment:
Minor: If you need to re-submit and you like the idea this can also be represented as follows:
`from soak_utils import (DDHHMMSS_format, add_pools, get_remote_logs,
    launch_rebuild, launch_snapshot, launch_exclude_reintegrate,
    create_ior_cmdline, cleanup_dfuse, create_fio_cmdline,
    build_job_script, SoakTestError)`

Reply:
i did try to make this change but my pylint in vscode did not like the format

Reply:
Not sure why pylint would in VS would report that. But Np. I just thought it looked cleaner that way.